README.md

```diff
@@ -1,28 +1,62 @@
 
-### Getting started
+### Strata application templates
+
+Application templates allow project teams to quickly spin up new production ready applications. Engineers can copy the templates to their own project's GitHub repository in order to get started more quickly. The templates provide many default technical decisions that represent best practices for implementing government systems. These best practices were evolved by Nava over many years of experience working on GovTech projects, and will continue to evolve. Strata application templates are available for Rails, Next.js, and Flask.
+
+**[The Rails template](https://github.com/navapbc/template-application-rails)** provides a template for web applications built with the Ruby on Rails framework. It has everything you need to build server-based web applications, including user-facing applications, APIs, and data engineering workflows. It comes integrated with the US Web Design System, adapters for AWS and Azure, and tools for supporting internationalization, authorization, automated testing, and more.
+
+**[The Next.js template](https://github.com/navapbc/template-application-nextjs)** provides a template for user-facing web applications built with the Next.js React framework. This template can be used to build traditional server-based web applications. It can also be used to build APIs. However, if your project has a significant data engineering (e.g. ETL jobs and other batch processing jobs) component, a Python-based tech stack is recommended (see the Python Flask template). This template can also be used to build static sites, but is currently not configured that way out of the box.
+
+**[The Flask template](https://github.com/navapbc/template-application-flask)** provides a template for APIs and offline processes built with Python Flask. This template can be used to build APIs. It can also be used to build simple user-facing applications. However, if your application is heavily on the frontend, you may want to consider using the Rails template or Next.js template since that comes already configured with internationalization, the US Web Design System, and Lookbook (in the Rails template) or Storybook (in the Next.js template).
+
+### Strata SDK
+
+[The Strata Software Development Kit (SDK)](https://github.com/navapbc/strata-sdk) will help your agency’s developers create best-in-class case management systems, including multi-page intake applications and case management workflows. The SDK is modular, allowing you to integrate the components you need with your program. This means you can build a new system from scratch, or plug new features into your existing system.
+
+### Strata CLI
+
+[The Strata CLI](https://github.com/navapbc/strata-cli) is a command line tool to make installing, upgrading, and using the Strata platform templates easy.
```

Comment:
Not the most critical, but wondering about the scope of the CLI with the rename. I think it may make sense to have a home for misc. utilities for accelerating development which could live as a part of this tool, but wondering if others agree? Some elements of Strata may not perfectly fit here, though it also doesn't make sense to name it something like `strata-everything-but-the-sdk-cli`. But if we imagine this CLI tool is only ever for interacting with the templates, should we rename to `strata-template-cli`?

And similar question as above for "Strata" and "Strata platform"? Maybe:
```suggestion
[The Strata CLI](https://github.com/navapbc/strata-cli) is a command line tool to make installing, upgrading, and using the Strata templates easy.
```